### ISSUE TITLE
Enable the ExistentialAny feature on all the repos

### ISSUE THREAD

**Issue by czechboy0** [apple/swift-openapi-generator#119]
Enable the ExistentialAny feature on all the repos.

The work was already done in PRs like #99, however we found that on Swift 5.8 toolchains there's an issue where the enable flag seems to propagate to dependees, so we will first disable the enforcement, until we require 5.9: #75.

**Comment by czechboy0** [comment]
More details on the issue: https://github.com/apple/swift-openapi-runtime/pull/26

And in https://github.com/apple/swift-openapi-runtime/pull/25#issuecomment-1635958854

**Comment by czechboy0** [comment]
We only disabled it on 5.8, but kept it enabled on 5.9.

So this issue tracks removing the conditional logic once we require 5.9, and have it enabled unconditionally.

**Comment by simonjbeaumont** [comment]
Do we feel this issue has any residual value since we have CI for >= 5.9 which tests that we are using _existential any_ in the whole codebase.

The only place we're not currently testing would be if we had code in an `#if swift(<5.9)` or similar, which we don't have.

Given we have #75, at some point we won't be using 5.8 at all, so I think this issue can probably be closed

**Comment by czechboy0** [comment]
Let me rename it, I meant to have it track the cleanup of the conditional logic, and block it on 5.9 adoption. We have a few more tasks like this, blocked on 5.9 adoption.

**Comment by simonjbeaumont** [comment]
OK, I thought that's what you wanted this issue to be for. I just thought we could just roll this into a task when we make 5.9 the minimum version.

**Comment by MahdiBM** [comment]
> however we found that on Swift 5.8 toolchains there's an issue where the enable flag seems to propagate to dependencies

For me personally, this doesn't cause an problems as long as the code itself is existential-any-compatible, which it is, but i'm curious to know what is the issue if it's filed anywhere 🤔

**Comment by czechboy0** [comment]
> > however we found that on Swift 5.8 toolchains there's an issue where the enable flag seems to propagate to dependencies
> 
> For me personally, this doesn't cause any problems as long as the code itself is existential-any-compatible, which it is, but i'm curious to know what is the issue if it's filed anywhere 🤔

Well, it seems to have been fixed in 5.9 between 04-26 and 04-27, unfortunately it still seems to cause issues in 5.8, which is why we only enable it on 5.9.

**Comment by czechboy0** [comment]
Landed in main, will be released in 1.0.0-alpha.1.
